## This Jupyter notebook answers HW04 questions for PHY 981 Nuclear Structure.

Author: Jacob Davison\
Date:   02/09/2021

### HW04 question 3

Use experimental data to obtain the b and c coefficients of the IMME for
the lowest energy 0+, T=1 states for A=30.

In [6]:
b = (243.681 - 255.620)/2
c = (251.282 - 243.681 - 255.620)/2

print('b = {:0.4f} MeV\nc = {:0.4f} MeV'.format(b,c))

b = -5.9695 MeV
c = -124.0095 MeV


We need to consolidate data from the nuclear chart, acccessible via [HERE](https://babwww.com/chart/chart-data.html). We will only gather data for which a $0^+$ T=1 is accessible. We look at isotopes along the diagonal line where A=30.

| Isotope | N | Z | T_z | BE (MeV) for $0^+,T=1$ state |
|---------|---|---|-----|------------------------|
|$^{30}P$ |15 |15 |0    |251.282                 |
|$^{30}S$ |16 |14 |1    |243.681                 |
|$^{30}Si$|14 |16 |-1   |255.620                 |

To compute the coefficients $b$ and $c$, we take the results in the table above and plug into Eq. 11.2 and 11.3.

$$b(0^+,T=1,A=30) = \frac{BE(T=1,T_z=1) - BE(T=1,T_z=-1)}{2}$$
$$c(0^+,T=1,A=30) = \frac{BE(T=1,T_z=0) - BE(T=1,T_z=1) - BE(T=1,T_z=-1)}{2}$$


We find that 

b = -5.9695 MeV\
c = -124.0095 MeV

### HW04 question 4

Use the liquid-drop model to estimate the excitation energy of the lowest
0+, T=4 state in 48Ti. Compare to experiment.

Lowest energy where T_z = T. We can estimate the excitation energy via the LDM, applying the isospin correction to the Coulomb term from IMME. This correction appears in Eq. 11.4. For charge radius, we estimate according to 

$$c_s = r_fA^{1/3}$$

where $r_f$ = 1.15 fm. This charge radius comes from the liquid drop model.

In [7]:
# LDM
def LDM(A, N, Z,T_z):
    T = 15.49*A
    S = 17.23*A**(2/3)
    V = 3/5*1.44/(1.15*A**(1/3))*(A**2/4-A*T_z+T_z**2) #MeV fm
    Sym = 22.6*(N-Z)**2/A
    
    return T - S - V- Sym

E = -LDM(208,126,82,4)
print('LDM excitation energy for 208Pb 0+ T=4, T_z=4 -- {:0.4f} MeV'.format(E))

print('Shift due to isospin = {:0.4f} MeV'.format(E+1636.430))

LDM excitation energy for 208Pb 0+ T=4, T_z=4 -- -1138.6719 MeV
Shift due to isospin = 497.7581 MeV


The excitation energy is shifted by approximately 500 MeV due to the isospin T=4. We compute the shift by subtracting the energy computed from LDM from the experimental energy.

### HW04 question 5

The isospin part of two-body Coulomb interaction between nucleons i and
j can be written as $(1 - \tau_{zi})(1- \tau_{zj})/4$. Rewrite this as a sum of terms
proportional to isospin operators tensor operators of rank 0, 1, and 2.

We can interpret the product $(1 - \tau_{zi})(1- \tau_{zj})/4$ as a tensor product of two tensor operators. Under the tensor product, two operators of ranks $N$ and $M$, $T^N \otimes T^M$, become a new tensor of rank $T^{N + M}$.

$$(1 - \tau_{zi})(1- \tau_{zj})\frac{1}{4} = (1\otimes 1 - 1\otimes\tau_{zj} - \tau_{zi}\otimes1 + \tau_{zi}\otimes\tau_{zj})\frac{1}{4}$$

The tensor product of a scalar operator (rank 0) and any other rank $N$ becomes $T^{0 + N}=T^N$. The scalar operator does not change the rank in the tensor product. Finally, we arrive at

$$(1 - \tau_{zi})(1- \tau_{zj})\frac{1}{4} = (1 - X^{(1)}_z + X^{(2)}_z)\frac{1}{4}$$

where, $X^{(1)} = \tau_{zi}+\tau_{zj}$ and $X^{(2)} = \tau_{zi}\otimes\tau_{zj}$.

### HW04 question 6

$X^{(2)} = [\tau_i \otimes \tau_j ]^2_0$ is two-body operator whose isospin dependence is given a by rank-2 tensor in isospin space, where $\vec{\tau} = 2\vec{t}$ i|s the single-particle isospin operator. Show that the matrix element $< T, T_z | X^{(2)} | T, T_z >$ gives a contribution the a and c terms of the IMME. One could add a term $dT^3_z$ to Eq. 11.1. Show that the rank-2 tensor operator gives d = 0. Hint - you will need Eq. 10.41.

According to Wigner Eckhart theorem,

$$< T, T_z | X^{(2)} | T, T_z > = (-1)^{T-T_z}\begin{pmatrix}T & 2 & T \\ -T_z & 0 & T_z\end{pmatrix}<T||X^{(2)}||T>$$

We compute the 3j coefficient according to Eq. 10.41

$$\begin{pmatrix}T & 2 & T \\ -T_z & 0 & T_z\end{pmatrix} = (-1)^{T-T_z}\frac{[3T_z^2 - T(T+1)]\delta_{T_zT_z}}{\sqrt{(2T-1)T(2T+1)(T+1)(2T+3)}}$$

Inserting this result into the expression for the matrix element, we find

$$< T, T_z | X^{(2)} | T, T_z > = \frac{[3T_z^2 - T(T+1)]}{\sqrt{(2T-1)T(2T+1)(T+1)(2T+3)}}<T||X^{(2)}||T>$$

The two body operator $X^{(2)}$ generates terms in $T_z^0$ and $T_z^2$, which contribute to the $a$ and $c$ coefficients in the IMME, respectively. Additionally, we immediately see that operators of rank, or below, $X^{(2)}$ cannot generate terms proportional to $T_z^3$ or higher order, and so we conclude that this rank 2 operator gives $d=0$ for the cubic contribution to the IMME.

### HW04 question 7

What is the kinetic energy for the protons in 208Pb in the Fermi gas model? What is the kinetic energy for the neutrons in 208Pb in the Fermi gas model? Estimate the total Coulomb interaction energy for 208Pb. Use these together with the experimental binding energy to get the total strong interaction
energy for 208Pb.

The average kinetic energy per proton, in the Fermi gas model, is (Eq. 12.21)

$$\frac{<T>_p}{Z} = \frac{3}{5}\epsilon_{fp}$$

where 

$$\epsilon_{fp} = \left(\frac{2Z}{A}\right)^{2/3}\epsilon_f$$

and $\epsilon_f$ = 36.7 MeV, according to the symmetric nuclear matter approximation.

Similarly,

$$\frac{<T>_n}{N} = \frac{3}{5}\epsilon_{fn}\\ \epsilon_{fn} = \left(\frac{2N}{A}\right)^{2/3}\epsilon_f$$

In [8]:
def T(n, A):
    epsf = 36.7 #MeV
    epsfn = (2*n/A)**(2/3)*epsf
    T = (3/5)*epsfn*n
    
    return T

print("""208Pb, Z = 82, N = 126

        <T>p = {:0.4f} MeV
        <T>n = {:0.4f} MeV""".format(T(82, 208), T(126,208)))

208Pb, Z = 82, N = 126

        <T>p = 1541.0551 MeV
        <T>n = 3153.1599 MeV


We estimate the Coulomb interaction energy via the liquid drop model. The liquid drop model (remembering E=-BE) defines

$$E = <T> + <V> = -16A+23\frac{(N-Z)^2}{A}$$

We can compute E via the LDM and then solve for $<V>$, inserting $<T> = <T>_n + <T>_p$:

$$<V> = -16(208)+23\frac{(126-82)^2}{208} - (<T>_p + <T>_n)$$

In [9]:
V = -16*208 + 23*(126-82)**2/208 - (T(82,208) + T(126,208))

print('Approximate Coulomb interaction V = {:0.4f} MeV'.format(V))

Approximate Coulomb interaction V = -7808.1381 MeV


To approximate the strong interaction energy, we can take the kinetic energy and Coulomb interaction energy we approximated above, and subtract the result from the experimental binding energy of 208Pb. 

In [10]:
T_tot = T(82, 208)+ T(126,208)
V = -7808.1381
E = T_tot + V

E_exp = -1636.430 

S = E-E_exp

print("Approximate strong interaction energy S = {:0.4f} MeV".format(S))

Approximate strong interaction energy S = -1477.4931 MeV
